src/libraries/System.Text.Json/tests/JsonElementParseTests.cs

```diff
@@ -58,12 +64,18 @@ public static void ParseValuePartialDataFail(string json)
         {
             var reader = new Utf8JsonReader(Encoding.UTF8.GetBytes(json));
 
+            Exception ex;
             try
             {
                 JsonElement.ParseValue(ref reader);
-                Assert.True(false, "Expected exception.");
+                ex = null;
             }
-            catch (JsonException) { }
+            catch (Exception e)
```

Comment:
Should this be:
```suggestion
            catch (JsonException e)
```

Reply:
It could be, but I followed the pattern in JsonDocumentTests, which I believe wants to fail with a nice Assert instead of a unhandled exception.